internal/engine/storage/page/page_bench_test.go

```diff
@@ -0,0 +1,63 @@
+package page
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+)
+
+const (
+	maxKeySize    = 100
+	maxRecordSize = 1_000
+)
+
+var (
+	pageSize = int(math.Round(2 * float64(Size) / 3)) // two-thirds of the max size
+)
+
+func generateRecordCell(b *testing.B) RecordCell {
+	keySize := 1 + rand.Intn(maxKeySize-1)
+	recordSize := 1 + rand.Intn(maxRecordSize-1)
+	key, record := make([]byte, keySize), make([]byte, recordSize)
+	_, err := rand.Read(key)
+	if err != nil {
+		b.Fatal(err)
+	}
```

Comment:
We tend to use `stretchr/testify` for tests.
You could easily rewrite this as
```
assert := assert.New(b)
...
_, err := rand.Read(key)
assert.NoError(err)
...
```
And you would get rid of a lot of err checks like this.